go.mod

```diff
@@ -32,15 +39,19 @@ require (
 	github.com/okzk/sdnotify v0.0.0-20180710141335-d9becc38acbd
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.2.1
+	github.com/prometheus/client_golang v1.0.0
```

Comment:
Something went wrong here. I recommend checking out `go.mod` from develop and just run  `go mod tidy` to get minimal update and if all looks good `make vendor`.